Sources/ViewControllers/Settings/SettingsTableViewController.swift

```diff
@@ -24,6 +29,10 @@ class SettingsTableViewController: UITableViewController {
             return "settings.tab-bar.item".localized
         } else if section == 1 {
```

Comment:
Will be great to create an enum for section instead to use an integer to improve the readability. What about something like that for enum and then use a switch case against?

`enum SettingsSection {
    case item 
    case information
}
`

Reply:
Thankyou for the review! 
Yes, I should create an enum as there will be more sections in the future, and it will be easy to navigate.

Reply:
@roger-tan can you review if it is correct?